modules/update-mirror-repository.adoc

```diff
@@ -114,6 +114,19 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
 
+... Obtain the image signature for the version that you are upgrading to from either link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] or link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)].
```

Comment:
We should remove the GCS link as it wont work without the full pull spec. It  also confuses the reader. cc @jiajliu @sdodson

Reply:
Agree, I suggested to drop this step some days ago in https://bugzilla.redhat.com/show_bug.cgi?id=1896679#c4